example/index.html

```diff
@@ -1,17 +1,35 @@
 <!DOCTYPE html>
-<head>
-  <title>react-p5-wrapper</title>
+<head lang="en">
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>@P5-wrapper/react</title>
```

Comment:
```suggestion
  <title>P5-wrapper/react</title>
```

Do we really need `@` here?

Reply:
I'm not sure about this change, since the package is still called `react-p5-wrapper`? WDYT?

Reply:
This is because of the NPM organisation. The names on GitHub and NPM are the same for the organisation, it just so happens that the name you install with is different (for now).

Both `npm install @P5-wrapper/react` and `npm install react-p5-wrapper` will be acceptable in the future since the current name will eventually just be an alias for the new one.

That's a whole other story though and will take time to implement and test properly. Perhaps this could be done as part of the version 4 release since it will be a major change version anyway. Let's see.

Reply:
That makes sense now and thanks for the explanation! So, it looks like we're all set 🏎️